apps/wing-console/console/app/demo/index.w

```diff
@@ -0,0 +1,117 @@
+bring cloud;
+// bring redis;
+
+let bucket = new cloud.Bucket();
+let queue = new cloud.Queue();
+let api = new cloud.Api();
+
+api.get("/test-get", inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
+  return cloud.ApiResponse {
+    status: 200,
+    body: Json.stringify({
+      query: Json (Json req).get("query"),
+    })
+  };
+});
+api.post("/test-post", inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
+  return cloud.ApiResponse {
+    status: 200,
+    body: "Hello, POST!"
+  };
+});
+
+let handler = inflight (message: str): str => {
+  bucket.put("hello.txt", "Hello, ${message}!");
+  log("Hello, ${message}!");
+  return message;
+};
+
+queue.setConsumer(handler);
+
+let counter = new cloud.Counter(initial: 0);
+new cloud.Function(inflight (message: str): str => {
+  counter.inc();
+  log("Counter is now ${counter.inc(0)}");
+  return message;
+}) as "IncrementCounter";
+
+let topic = new cloud.Topic() as "Topic";
+topic.onMessage(inflight (message: str): str => {
+  log("Topic subscriber #1: ${message}");
+  return message;
+});
+topic.onMessage(inflight (message: str): str => {
+  log("Topic subscriber #2: ${message}");
+  return message;
+});
+
+// let r = new redis.Redis();
+// new cloud.Function(inflight (message :str) :str => {
+//   log("${r.url()}");
+//   r.set("wing", message);
+//   let value = r.get("wing");
+//   log("${value}");
+//   return r.url();
+// }) as "Redis interaction";
+
+
+let table = new cloud.Table(cloud.TableProps{
+  name: "simple-table",
+  primaryKey: "id",
+  columns: {
+    id: cloud.ColumnType.STRING,
+    name: cloud.ColumnType.STRING,
+    date: cloud.ColumnType.DATE,
+    active: cloud.ColumnType.BOOLEAN,
+  },
+});
+
+let rateSchedule = new cloud.Schedule(cloud.ScheduleProps{
+  rate: 5m
+}) as "Rate Schedule";
+
+rateSchedule.onTick(inflight () => {
+  log("Rate schedule ticked!");
+});
+
+let cronSchedule = new cloud.Schedule(cloud.ScheduleProps{
+  cron: "* * * * ?"
+}) as "Cron Schedule";
+
+// cronSchedule.onTick(inflight () => {
```

Comment:
and schedule?

Reply:
Same as https://github.com/winglang/wing/pull/3079#discussion_r1242166070: this is how the file is in the previous repository.